### ISSUE TITLE
References gone when using Output Directory 

### ISSUE THREAD

**Issue by kmisiunas** [thomasjo/atom-latex#129]
References do not appear in compiled PDF when the using "Output Directory" option. Other parameters:

MAC OS X 10.11
latexmk
lualatex

**Comment by thomasjo** [comment]
Can you provide a minimal example that reproduces the problem? I use an output directory and I'm not able to reproduce this.

**Comment by kmisiunas** [comment]
So i had a look for the cause. It looked like the folder name "/Entropic Splitter, 2014/" was causing the problem because I could compile the file if it was placed before it. Then I renamed the folder to "tmp" and it worked. Now the weird part. Once I renamed the folder back to  "/Entropic Splitter, 2014/" the error was gone. Could not work out the exact cause. Maybe a special character? 

Thanks for the help. If I encounter this again, I will post it here.
